scull/Makefile

```diff
@@ -1,5 +1,5 @@
 # Comment/uncomment the following line to disable/enable debugging
-#DEBUG = y
```

Comment:
@jypan-ccu thanks for the PR!

Did you intend to turn all of the DEBUG flags on?  My thought is that we leave them off by default and users can turn them on if they need them for debugging.

Reply:
That's OK! I just make sure all the code segment works under DEBUG flags on. We should leave them off by default.
I have updated the pull request.